_layouts/post.html

```diff
@@ -0,0 +1,56 @@
+<script src="https://apis.google.com/js/platform.js" async defer></script>
+<script src="https://apis.google.com/js/plusone.js"></script>
+<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
+{% include header.html %}
+<div class="container-fluid container-body">
+  <div class="col-md-8 col-md-offset-2 col-centered">
+    <section>
+      <div class="post-header">
+        {% if page.pagename != nil %}
+        <h2>{{ page.pagename }}</h2>
+        {% else %}
+        <h2>{{ page.title }}</h2>
+        {% endif %}
+        <p class="metadata">
+          <span class="author">{{ page.author }}</span> - <span class="date">{{ page.date | date: "%-d %B %Y" }}</span>
+        </p>
+      </div>
+
+      <hr/>
+
+      <div class="post-content" id="doc">
```

Comment:
One minor downside about using a `doc` element is that it sets a minimum height of 500px, might be better to just have the attribute(s) we need for blogs?